src/PgManyToManyRelationInflectionPlugin.ts

```diff
@@ -47,11 +47,15 @@ export const PgManyToManyRelationInflectionPlugin: GraphileConfig.Plugin = {
           rightTable,
           junctionTable,
         } = processDetails(details);
-        if (
-          typeof junctionRightRelation.extensions?.tags.manyToManyFieldName ===
-          "string"
-        ) {
-          return junctionRightRelation.extensions.tags.manyToManyFieldName;
+        const override =
+          junctionRightRelation.extensions?.tags.manyToManyConnectionFieldName;
+        if (typeof override === "string") {
+          return this.camelCase(override);
```

Comment:
Since we're changing this anyway, I think now would be a good time to remove the `this.camelCase`. Time and again people have wanted ways of fixing bugs in the camelCase inflector, so when they explicitly state `FieldName` that should be seen as the explicit name for the field and it should be used verbatim.

We should still validate that it's a reasonable GraphQL name though:

```suggestion
          return this.coerceToGraphQLName(override);
```